plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/AbstractKuduConnectorTest.java

```diff
@@ -544,10 +630,75 @@ public void testWrittenStats()
     }
 
     @Override
-    public void testReadMetadataWithRelationsConcurrentModifications()
+    protected void testReadMetadataWithRelationsConcurrentModifications(int readIterations, int testTimeoutSeconds)
+            throws Exception
     {
-        // TODO Support these test once kudu connector can create tables with default partitions
-        throw new SkipException("TODO");
+        // TODO Remove this overriding test once kudu connector can create tables with default partitions
+        Stopwatch testWatch = Stopwatch.createStarted();
+
+        int readerTasksCount = 6
+                + (hasBehavior(SUPPORTS_CREATE_VIEW) ? 1 : 0)
```

Comment:
Please avoid `hasBehavior()` in sub class since the test class already knows the behavior.

Reply:
@ebyhr Got it! The entire code block is eliminated after the update, but I'll also avoid it next time.